src/components/Navigation/Navigation.tsx

```diff
@@ -1,15 +1,34 @@
 import React, { Component, Fragment } from 'react';
 import styled from 'styled-components';
-import { Menu, MenuButton, MenuList } from '@chakra-ui/menu';
+import { Menu, MenuButton, MenuList, MenuItem } from '@chakra-ui/menu';
 import { AtSignIcon, StarIcon, ChevronDownIcon } from '@chakra-ui/icons';
 import { Button } from '@chakra-ui/button';
 import { NavigationSubItem } from './NavigationSubItem';
 
-const StyledNavigation = styled.nav`
-  display: flex;
-  align-self: stretch;
-  margin-left: 1rem;
-  color: var(--color-white);
+// const StyledNavigation = styled.nav`
```

Comment:
delete any commented out code prior to PR

Reply:
Yes Arnab, no problem, I am just leaving it so when I will have a chat with Otavio I have some questions regarding these commented out code.